pkgs/top-level/all-packages.nix

```diff
@@ -33523,6 +33523,8 @@ with pkgs;
 
   DisnixWebService = callPackage ../tools/package-management/disnix/DisnixWebService { };
 
+  i3a = callPackage ../misc/i3a { };
```

Comment:
I'm not sure if this should go in `../applications/window-managers/i3`. There is some unofficial stuff in there like this package.